tests/helpers/MockToolRegistry.ts

```
@@ -163,14 +181,14 @@ export class MockToolRegistry {
             this.dataLayer.removeAgentFromRoom(args.roomName, args.agentName);
             return { content: [{ type: 'text', text: JSON.stringify({ success: true }) }] };
           
-          case 'agent_communication/list_room_users':
+          case 'agent_communication_list_room_users':
             if (!this.dataLayer.roomExists(args.roomName)) {
               throw new RoomNotFoundError(args.roomName);
             }
             const agents = this.dataLayer.getRoomAgents(args.roomName);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)